Makefile

```diff
@@ -232,6 +232,7 @@ artifacts: kustomize
 	if [ -d artifacts ]; then rm -rf artifacts; fi
 	mkdir -p artifacts
 	$(KUSTOMIZE) build config/default -o artifacts/manifests.yaml
+	$(KUSTOMIZE) build config/dev -o artifacts/manifests-dev.yaml
```

Comment:
there seems to be something wrong with the rebase

Reply:
Fixed.